### PR TITLE
Added an example script for creating a workload

### PR DESCRIPTION
**Pull request by fattyuke**
Added another example script for creating a workload based on an
existing template.

### CODE REVIEW

File: examples/create-workload.rb [hunk 1]
```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env ruby
+
+#
+# Script Name:: create-workload
+# Script Type: API script, using RubySDK and Bracket FOG plugin
+# Script Description: Create a workload based on Workload Templates
+#
+# Script dependancies:
```
Comment (bburtin) [inline examples/create-workload.rb]: typo

Comment (fattyuke) [inline examples/create-workload.rb]: Thanks Boris. Edited my comment ;)

File: examples/create-workload.rb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+##########################################################
+# Deploy Workload based on selected template
+# Use the default billing group
+##########################################################
+# You can create a hash of attribute values that you would like to include
+# in the workload deployment. Below is an example. Please feel free to 
+# change any of these values
+attributes = Hash.new
+attributes["name"]=myName
```
Comment (Kuzmin) [inline examples/create-workload.rb]: code style: there should be spaces around operators (in this case the `=`)

**Comment by Kuzmin** [comment]
You seem to be mixing tabs and spaces in this PR, the rest of the codebase uses 2 spaces for indentation, so we should try adhering to that.

**Comment by fattyuke** [comment]
Thanks for the feedback. I'm just hacking my way through this and learning it at the same time. I have an operational question. Should I close the pull request, make the suggested changes then create a new pull request?

**Comment by Kuzmin** [comment]
I would probably make the changes, do `git commit --amend`, and then force push. This would be one of the few legitimate use-cases for a force-push, I'd say.

Pinging @bburtin for his opinions on force pushes.

**Comment by bburtin** [comment]
The only problem with amending is that you lose comments that were made about the previous commit.  Kind of a lame limitation with the pull request model.  Not a big issue in this case.  Either amend the initial commit like Alex says, or do a separate commit that addresses our comments.  Either way is fine.

**Comment by fattyuke** [comment]
Thanks Boris. I made my changes and did an amend and a force push like Alex suggested. I'm assuming this PR will just pick up the new file with the changes.

**Comment by Kuzmin** [comment]
This LGTM

**Pull request merged**